### DISCUSSION TITLE
Add the ability to pin and unpin aliases in the Android mobile app

### DISCUSSION THREAD

**Discussion by nopapersplease** [simple-login/app#740] [Feature Requests]
The Android mobile app does not appear to have the ability to pin/unpin an alias. I've scoured and searched each screen and I can't find a button on an alias record to pin/unpin it. The only way that I can find to pin/unpin an alias is to log into the desktop web client interface. Please add this feature. Thanks.

**Comment by ntnhon** [comment]
Hello, I answered the question here: https://github.com/simple-login/app/discussions/739#discussioncomment-1910771
